R/extract.R

```diff
@@ -9050,3 +9050,96 @@ extract.hurdle <- extract.zeroinfl
 #' @export
 setMethod("extract", signature = className("hurdle", "pscl"),
           definition = extract.hurdle)
+
+# -- extract.logitr (logitr) ---------------------------------------------------
+
+#' @noRd
+extract.logitr <- function(
+  model,
+  include.nobs   = TRUE,
+  include.loglik = TRUE,
+  include.aic    = TRUE,
+  include.bic    = TRUE,
+  ...
+) {
+  summary <- summary(model)
+
+  coefnames <- names(summary$coefficients)
+  coefs <- summary$coefTable$Estimate
+  se <- summary$coefTable$`Std. Error`
+  pval <- summary$coefTable$`Pr(>|z|)`
+
+  n <- summary$n$obs
+  ll <- summary$logLik
+
+  gof <- numeric()
+  gof.names <- character()
+  gof.decimal <- logical()
+  if (include.nobs == TRUE) {
+    gof <- c(gof, n)
+    gof.names <- c(gof.names, "Num. obs.")
+    gof.decimal <- c(gof.decimal, FALSE)
+  }
+  if (include.loglik == TRUE) {
+    gof <- c(gof, ll)
+    gof.names <- c(gof.names, "Log Likelihood")
+    gof.decimal <- c(gof.decimal, TRUE)
+  }
+  if (include.aic == TRUE) {
+    gof <- c(gof, summary$statTable["AIC", ])
+    gof.names <- c(gof.names, "AIC")
+    gof.decimal <- c(gof.decimal, TRUE)
+  }
+  if (include.bic == TRUE) {
+   gof <- c(gof, summary$statTable["BIC", ])
+    gof.names <- c(gof.names, "BIC")
+    gof.decimal <- c(gof.decimal, TRUE)
+  }
+
+  tr <- createTexreg(
```

Comment:
Same as above: Format in line with other calls of `createTexreg` in the package.